### PR TITLE
Allow dependabot to update GitHub actions

### PR DESCRIPTION
**Pull request by kingthorin**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
```
Comment (thc202) [inline .github/dependabot.yml:3]: Remove the comments please.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```
Comment (thc202) [inline .github/dependabot.yml:6]: `"monthly"`

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    groups:
+      dependencies:
```
Comment (thc202) [inline .github/dependabot.yml:8]: `gha` or something that's obvious this is updating the GitHub Actions.

Comment (kingthorin) [reply .github/dependabot.yml:8]: Line 3?

Comment (kingthorin) [reply .github/dependabot.yml:8]: Or do you mean for the commit-message or prefix?

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message

Comment (thc202) [reply .github/dependabot.yml:8]: I mean this line, which is the name of the group (and used for commit too).

**Comment by kingthorin** [comment]
Assuming this is what we want elsewhere as well I'll update them after this is merged.

**Review by ricekot — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**